libr/core/agraph.c

```diff
@@ -4858,6 +4858,7 @@ R_API int r_core_visual_graph(RCore *core, RAGraph *g, RAnalFunction *_fcn, int
 	}
 	r_config_hold_restore (hc);
 	r_config_hold_free (hc);
+	showcursor(core, true);
```

Comment:
add missing space, and only do this if color is set

Reply:
Regarding the space, there wasn't a space before either but I'll add it.
Regarding the color. I'm not sure what you mean or how to do that. I merely looked for the place where the cursor is set to hidden and after the graph view closes I added a line to make it visible (it was made invisible earlier in the function).
I can't say I understand the project, merely was able to mimic the existing logic and thought to save someone some time fixing the issue I opened a little before.